Fix and enhance auth config chapter (#5971)
qgis/QGIS-Documentation: docs/user_manual/auth_system/auth_overview.rst

   Utilities menu

* **Input master password**: opens the master password input dialog, independent 
  of performing any authentication database command.
* **Clear cached master password**: unsets the master password if it has been 
  set via input dialog
* **Reset master password**: opens a dialog to change the master password (the 
  current password must be known) and optionally back up the current database
* **Clear network authentication access cache**: clears the authentication cache
  of all connections
* **Automatically clear network authentication access cache on SSL errors**: the

>   connection cache stores all authentication connections data even when the 
>   connection fails. If you make any change to the authentication configurations 
>   or to the certification authorities, you should clear the authentication cache 

Reviewer: ```suggestion
  connection cache stores all authentication data for connections, also when the
  connection fails. If you change authentication configurations or certification authorities,
  you should clear the authentication cache 
```

  or restart QGIS. When this option is checked, the authentication cache will be
  automatically cleared every time an SSL error occurs and you choose to abort 
  the connection
* **Integrate master password with your Wallet/Keyring**: adds the master 
  password to your personal Wallet/Keyring
* **Store/update the master password in your Wallet/Keyring**: updates the 
  changed master password in your Wallet/Keyring
* **Clear the master password from your Wallet/Keyring**: deletes the master
  password from your Wallet/Keyring
* **Enable password helper debug log**: enables a debug tool that will contains
  all the log information of the authentication methods
* **Clear cached authentication configurations**: clears the internal lookup cache

>   for configurations used to speed up network connections. This does not clear

Reviewer: ```suggestion
  for configurations, used to speed up network connections. This does not clear
```
?

  QGIS’s core network access manager’s cache, which requires a relaunch of QGIS.
* **Remove all authentication configurations**: clears the database of all
  configuration records, without removing other stored records.
* **Erase authentication database**: schedules a backup of the current database

>   and complete rebuild of the database table structure. These actions are
>   scheduled for a later time, so as to ensure other operations like project
>   loading do not interrupt the operation or cause errors due to a temporarily

Reviewer: ```suggestion
  and complete rebuild of the database table structure. The actions are
  scheduled for a later time, to ensure that other operations, like project
  loading, do not interrupt the operation or cause errors due to a temporarily
```
